u6-spas/src/views/Converter.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+ <table>
+   <tr>
+     <td><input v-model="input" type="number"></td>
+     <td>=</td>
+     <td><span class="output">{{output.toFixed(2)}}</span></td>
+   </tr>
+  <tr>
+      <td>
+        <select v-model="inputUnit">
+          <option v-for="unit in units" :key="unit.id" :value="unit.id">
+              {{unit.label}}
+          </option>
+        </select>
+      </td>
+      <td class="fake-link" @click="swap()">swap</td>
+      <td>
+        <select v-model="outPutUnit">
```

Comment:
ฉันคิดว่าคุณพิมพ์ผิด... `outPutUnit` กับ `outputUnit` คุณแน่ใจหรือว่าวิธีนี้ได้ผล? คุณทดสอบหรือไม่
I think you have a typo here... `outPutUnit` vs `outputUnit`. Are you sure this works? Did you test it?